### ISSUE TITLE
Display real link when hovering over masked links

### ISSUE THREAD

**Issue by HybridEidolon** [ajbura/cinny#1267]
### Describe the problem

When reading a message in a room that contains a masked link (i.e. Markdown text hyperlink), it is not possible to know where the anchor _actually_ links to in the desktop application, as there is no default browser URL display. This can lead to confusion, or worse, deception.

### Describe the solution you'd like

There should be a tooltip on hovering over a masked link, or any link, which shows some or all of the actual hyperlink's URL. This allows the user to be informed of the actual URL that they are about to open.

### Alternatives considered

- Instead of opening the anchor's URL directly, display a warning prompt when clicking that presents the full URL of the anchor.
- Display some inline indicator when an anchor link is masked.

**Comment by u9000-Nine** [comment]
This is also an issue on the PWA version (at least in Helium)
